.github/workflows/ci.yml

```
@@ -14,16 +14,26 @@ jobs:
 
     steps:
       - name: Setup Node.js
-        uses: actions/setup-node@v2.1.2
+        uses: actions/setup-node@v3.1.1
```

```suggestion
        uses: actions/setup-node@v2.1.2
```
@ospencer let's keep this pinned until we want to do a full upgrade. I think our other workflows still have the v2